includes/amp-helper-functions.php

```diff
@@ -632,6 +632,32 @@ function amp_get_content_sanitizers( $post = null ) {
 		$post = null;
 	}
 
+	$sanitizer_classes = array(
+		'AMP_Core_Theme_Sanitizer',
+		'AMP_Img_Sanitizer',
+		'AMP_Form_Sanitizer',
+		'AMP_Comments_Sanitizer',
+		'AMP_Video_Sanitizer',
+		'AMP_O2_Player_Sanitizer',
+		'AMP_Audio_Sanitizer',
+		'AMP_Playbuzz_Sanitizer',
+		'AMP_Embed_Sanitizer',
+		'AMP_Iframe_Sanitizer',
+		'AMP_Gallery_Block_Sanitizer', // Note: Gallery block sanitizer must come after image sanitizers since itś logic is using the already sanitized images.
+		'AMP_Block_Sanitizer', // Note: Block sanitizer must come after embed / media sanitizers since it's logic is using the already sanitized content.
+		'AMP_Script_Sanitizer', // Note: This whitelist sanitizer must come at the end to clean up any remaining issues the other sanitizers didn't catch.
+		'AMP_Style_Sanitizer', // Always penultimate.
+		'AMP_Tag_And_Attribute_Sanitizer', // Always ultimate.
+	);
+
+	// @todo This doesn't help for classes that get added via amp_content_sanitizers. Does that matter?
```

Comment:
In practice this probably doesn't matter. If someone is adding a new sanitizer, they'd be doing so via a filter anyway. They would just be required to supply the defaults when they do so. For example:

```php
add_filter( 'amp_content_sanitizers', function( $sanitizers ) {
    require_once __DIR__ . '/class-sanitizer.php'
    return array_merge(
        $sanitizers,
        array(
            '\Foo\Sanitizer' => Sanitizer::get_default_args(),
        )
    );
} );
```